libraries/freertos_plus/standard/tls/src/iot_tls.c

```diff
@@ -141,6 +141,9 @@ typedef struct TLSContext
 
 #define TLS_PRINT( X )    configPRINTF( X )
 
+BaseType_t prvDefault_DateIsInThePast(BaseType_t day, BaseType_t month, BaseType_t year);
+dateIsInThePast_t pDateIsInThePast = prvDefault_DateIsInThePast;
```

Comment:
Could this callback be part of the TLSContext_t and be set in TLS_Init? That would allow us to not have this in the global scope.

Reply:
It could be, but that means that every TLS context will need to be initialized to the application date checker at run-time.  The default is only useful for removing the requirement for NULL checks.

Reply:
If we change the signature of TLS_Init then it will break the existing demo's.  We can certainly add this to the context but it still seems strange to keep an independent copy in each context when they should all be identical.

Reply:
That's a good point. Lets keep this as you have it now then.